lib/to-regexp.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function(source) {
+  if (Array.isArray(source)) {
+    return source.map(regexpify);
+  } else {
+    return regexpify(source);
+  }
+};
+
+function regexpify(source) {
+  if (typeof source === 'string') {
+    if (!source.length) {
+      throw new Error('`ignorePattern` is empty');
```

Comment:
This should probably be a more generic error message as this is used for both `ignorePattern` and `utilityPattern`